### ISSUE TITLE
When having a select inside a CRUD_field it only fetches select items if there's another select with the same query in the main form (not inside the CRUD_field). 

### ISSUE THREAD

**Issue by kingmatusevich** [perak/kitchen-cli#9]
Without a pointless Select in the main form with the same query: 
(a) the main form
![screen shot 2016-01-12 at 02 40 28](https://cloud.githubusercontent.com/assets/1448511/12255807/7f63998a-b8d6-11e5-832f-d10c5209cfe0.png)
(b) the CRUD with a select that doesn't show options. 
![screen shot 2016-01-12 at 02 40 36](https://cloud.githubusercontent.com/assets/1448511/12255816/953020d0-b8d6-11e5-8bd3-d94a38497342.png)

With a pointless Select in the main form with the same query:
(a) the main form 
![screen shot 2016-01-12 at 02 42 35](https://cloud.githubusercontent.com/assets/1448511/12255822/a5fc6b9e-b8d6-11e5-8c90-25b1f94938f6.png)
(b) the CRUD with a select that now actually shows options. 
![screen shot 2016-01-12 at 02 42 41](https://cloud.githubusercontent.com/assets/1448511/12255828/b84e30fc-b8d6-11e5-9a4f-2e370236419c.png)

Also, I'm pretty sure the joins don't work in a CRUD_field, even with the useless select. 
Thank you for your great work! 😊

**Comment by kingmatusevich** [comment]
![screen shot 2016-01-12 at 02 50 41](https://cloud.githubusercontent.com/assets/1448511/12255882/4c0d19f2-b8d7-11e5-82dd-a6107804e9a7.png)
Here you can see what I meant when I said the joins don't work.

**Comment by perak** [comment]
@kingmatusevich 

OK, this is two issues:

First one is because "kitchen" didn't automatically created subscription for "product" required by CRUD's insert form. Workaround is to refer that query in your page (page that contains CRUD) in `related_queries`. For example:

```
  "related_queries": [
    { "name": "product_query_name", "params": [ ] }
  ]
```

Second issue: CRUD currently cannot do what you are trying to implement - it's purpose is to enter simple data (not data from another collection) into array. However, it will be great to have this feature and I'll try to implement it.

**Comment by kingmatusevich** [comment]
Great, glad to hear you are on top of it, I obviously think this is an awesome feature to have as well. Let me know if I can help you somehow.
Good work :+1:

**Comment by kingmatusevich** [comment]
I can confirm the workaround you posted (`related_queries`) does fix my issue with the subscription.

**Comment by kingmatusevich** [comment]
Of course, the CRUD doesn't traverse the Join I defined with the productId and the rest of the product fields.
